R/tests/testthat/test_diagnostics.R

```diff
@@ -150,28 +151,31 @@ test_that("performance_metrics", {
   )
 })
 
-test_that("rolling_mean", {
+test_that("rolling_mean_or_median", {
   skip_if_not(Sys.getenv('R_ARCH') != '/i386')
   x <- 0:9
   h <- 0:9
-  df <- prophet:::rolling_mean_by_h(x=x, h=h, w=1, name='x')
+  agg_args <- c('mean', 'median')
+  for (agg in agg_args){
```

Comment:
i don't particularly like doing it this way but not sure if there is an option to `parametrize` with `testthat` ?